config-vm.md

```diff
@@ -0,0 +1,49 @@
+# <a name="VirtualMachineSpecificContainerConfiguration" /> Virtual-machine-specific Container Configuration
+
+Virtual-machine-based containers require additional configuration to that specified in the [base spec configuration](config.md).
+
+This **optional** configuration is specified in a "VM" object:
+
+* **`imagePath`** (string, REQUIRED) host path to file that represents the root filesystem for the container virtual machine.
```

Comment:
It (**`imagePath`**) should be optional.

**`imagePath`** and **`initrd`** are both booting media. We need to mark them both optional and document that at least one of them should be specified.

Moreover, since **`imagePath`** and **`initrd`** are both booting media, they are recommended to be put together. So we can move **`imagePath`** to the **`kernel`** section around the **`initrd`** field. After we do it, we may also need to rename **`kernel`** to **`boot`** and rename `kernel.path` to `boot.kernel`. Another reason why **`boot`** is better than **`kernel`** here is that **`bios`** or **`cbfs`** can be added to the **`boot`** section if any team strongly require them.

Finally, I like the name **`imagePath`**, but I didn't expect that it has ambiguous with the **`path`** field in the [Root Configuration](config.md#Root-Configuration) section. So I suggest to rename it to **`initImage`**, since it and **`initrd`** are almost the same functionality.